Owl/app/src/androidTest/java/com/example/owl/ui/NavigationTest.kt

```diff
@@ -115,7 +115,7 @@ class NavigationTest {
 
     private fun getOnboardingFabLabel(): String {
         return InstrumentationRegistry.getInstrumentation().targetContext.resources
-            .getString(R.string.continue_to_courses)
```

Comment:
- Remove the activity property
- Remove the onActivity
- Replace `activity` with `composeTestRule.activity`
- Replace `InstrumentationRegistry.getInstrumentation().targetContext.resources` with `composeTestRule.activity.resources`

Reply:
Thanks! Done.